src/MembershipsSDKAdapter.js

```diff
@@ -112,6 +113,7 @@ export default class MembershipsSDKAdapter extends MembershipsAdapter {
       orgID: deconstructHydraId(membership.personOrgId).id,
       muted: null,
       sharing: null,
+      guest: null,
```

Comment:
add it at the end of the list, like in the interface definition, unless there's a specific reason to do otherwise

Reply:
Changed